webapp/sass/responsive/_mobile.scss

```diff
@@ -209,11 +209,12 @@
 
         .post__header {
             margin-bottom: 0;
+            padding-right: 50px;
```

Comment:
@kaifaust Try the message details view, in the message details view the date stamp is much larger.